topeft/modules/object_selection.py

```diff
@@ -42,44 +42,60 @@ def iseTightTau(idDeepTauVSe):
-    elif (year == "2016APV"):
-        wploose  = get_tc_param("btag_wp_loose_UL16APV")
-        wpmedium = get_tc_param("btag_wp_medium_UL16APV")
-    elif (year == "2017"):
-        wploose  = get_tc_param("btag_wp_loose_UL17")
-        wpmedium = get_tc_param("btag_wp_medium_UL17")
-    elif (year == "2018"):
-        wploose  = get_tc_param("btag_wp_loose_UL18")
-        wpmedium = get_tc_param("btag_wp_medium_UL18")
-    elif (year == "2022"):
-        wploose = get_tc_param("btag_wp_loose_2022")
-        wpmedium = get_tc_param("btag_wp_medium_2022")
-    elif (year == "2022EE"):
-        wploose = get_tc_param("btag_wp_loose_2022EE")
-        wpmedium = get_tc_param("btag_wp_medium_2022EE")
-    elif (year == "2023"):
-        wploose = get_tc_param("btag_wp_loose_2023")
-        wpmedium = get_tc_param("btag_wp_medium_2023")
-    elif (year == "2023BPix"):
-        wploose = get_tc_param("btag_wp_loose_2023BPix")
-        wpmedium = get_tc_param("btag_wp_medium_2023BPix")
+    if btagger == "btagDeepFlavB":
+        if (year == "2016"):
+            wploose  = get_tc_param("btag_wp_loose_UL16")
+            wpmedium = get_tc_param("btag_wp_medium_UL16")
+        elif (year == "2016APV"):
+            wploose  = get_tc_param("btag_wp_loose_UL16APV")
+            wpmedium = get_tc_param("btag_wp_medium_UL16APV")
+        elif (year == "2017"):
+            wploose  = get_tc_param("btag_wp_loose_UL17")
+            wpmedium = get_tc_param("btag_wp_medium_UL17")
+        elif (year == "2018"):
+            wploose  = get_tc_param("btag_wp_loose_UL18")
+            wpmedium = get_tc_param("btag_wp_medium_UL18")
+        elif (year == "2022"):
+            wploose = get_tc_param("btag_wp_loose_2022")
+            wpmedium = get_tc_param("btag_wp_medium_2022")
+        elif (year == "2022EE"):
+            wploose = get_tc_param("btag_wp_loose_2022EE")
+            wpmedium = get_tc_param("btag_wp_medium_2022EE")
+        elif (year == "2023"):
+            wploose = get_tc_param("btag_wp_loose_2023")
+            wpmedium = get_tc_param("btag_wp_medium_2023")
+        elif (year == "2023BPix"):
+            wploose = get_tc_param("btag_wp_loose_2023BPix")
+            wpmedium = get_tc_param("btag_wp_medium_2023BPix")
+    elif btagger == "btagPNetB":
+        if (year == "2022"):
+            wploose = get_tc_param("btag_wp_loose_PNet_2022")
+            wpmedium = get_tc_param("btag_wp_medium_PNet_2022")
+        elif (year == "2022EE"):
+            wploose = get_tc_param("btag_wp_loose_PNet_2022EE")
+            wpmedium = get_tc_param("btag_wp_medium_PNet_2022EE")
+        elif (year == "2023"):
+            wploose = get_tc_param("btag_wp_loose_PNet_2023")
+            wpmedium = get_tc_param("btag_wp_medium_PNet_2023")
+        elif (year == "2023BPix"):
+            wploose = get_tc_param("btag_wp_loose_PNet_2023BPix")
+            wpmedium = get_tc_param("btag_wp_medium_PNet_2023BPix")
```

Comment:
We could probably simplify this block by just adding the year string to the end of things like `btag_wp_medium_PNet_`. I won't hold up this PR, but if you want you can open an issue.